src/cpp/src/lora_adapter.cpp

```diff
@@ -115,19 +115,14 @@ struct AutoSafetensor: public safetensors_File {
     }
 };
 
-
-// Reads a file with a given filename expecting Safetensors file format.
-// The data is read to a solid memory block and the function returns a map of OV Constants allocated on top of that block.
 // The key in the map is a tensor name and the Constant uses a region of memory from the memory block.
 // Each Constant holds a shared pointer to the block in the runtime info.
 // The memory block will be deallocated when the last Constant is destroyed.
-ConstantMap read_safetensors(const std::filesystem::path& filename) {
-    auto buffer = read_file_helper(filename);
+ConstantMap safetensors_init(const ov::Tensor & safetensor) {
```

Comment:
Minor, the name is not so clear for understanding.... Is it possible to name smt like:
* `ov_tensor_to_constant_map`
* `safetensor_to_constant_map`..